Scripts/localize.py

```diff
@@ -134,9 +134,9 @@ def localize(path, language, include_pods_and_frameworks):
 
     # TODO: This is super ugly, we have to come up with a better way of doing it
     if include_pods_and_frameworks:
-        find_cmd = 'find . ../Pods/WordPress* ../Pods/WPMediaPicker ../WordPressShared/WordPressShared ../Pods/Gutenberg -name "*.m" -o -name "*.swift" | grep -v Vendor | grep -v ./WordPressTest/I18n.swift'
+        find_cmd = 'find . ../Pods/WordPress* ../Pods/WPMediaPicker ../WordPressShared/WordPressShared ../Pods/Gutenberg -name "*.m" -o -name "*.swift" | grep -v Vendor | grep -v ./WordPressTest/I18n.swift | grep -v ./WordPressStatsWidgets/Views/Localization/LocalizedStringKey+extension.swift'
```

Comment:
nit: why use `grep -v` instead of adding finer predicates to `find`, i.e. `-not -name Vendor -not -path ./WordPressTest/I18n.swift -not -path ./WordPressStatsWidgets/Views/Localization/LocalizedStringKey+extension.swift` (or something like this – not tested)

Reply:
Yes – that would certainly be an improvement 😅 

I figured following the existing convention was the least-impactful change, and given that this file is going away anyway soon (to be replaced by a Ruby version) I figured it'd be easiest to review this way.

But yes, it _is_ a bit of an anti-pattern, no doubt!